docs/process/labels.md

```diff
@@ -3,40 +3,47 @@
 This section outlines the labels currently used in the project, and adds context
 about what each label represents.
 
-## Common labels
+## Areas of Work
 
 These labels are used for both issues and pull requests, and are assigned to
-help understand the type of work involved.
-
-|                               | Label name       | Description |
-| ----------------------------- | -----------------| ----------- |
-| [:mag_right:][docs]           | `docs`           | Issues and pull requests related to documentation work on the project |
-| [:mag_right:][infrastructure] | `infrastructure` | Issues and pull requests related to scripts and tooling for GitHub Desktop |
-| [:mag_right:][tech-debt]      | `tech-debt`      | Issues and pull requests related to addressing technical debt or improving the codebase |
-
-## Issue-specific labels
-
-This section is organized into sub-groups, as groups of labels relate to
-specific work being done in the issue tracker.
-
-### Issue triage
+help identify what area a task is associated with. This helps organize upcoming
+work and enables contributors to focus on a specific area of the project that
+they are passionate about.
+
+|                                       | Label name               | Description |
+| ------------------------------------- | -------------------------| ----------- |
+| [:mag_right:][area:accessibility]     | `area:accessibility`     | Accessibility issues or improvements in the product |
+| [:mag_right:][area:dependencies]      | `area:dependencies`      | Upgrading dependencies that the project uses |
+| [:mag_right:][area:diffs]             | `area:diffs`             | Diff rendering and our use of [CodeMirror](https://codemirror.net/) |
+| [:mag_right:][area:docs]              | `area:docs`              | Documentation for the project |
+| [:mag_right:][area:electron]          | `area:electron`          | How we use [Electron](https://electronjs.org) in the product |
+| [:mag_right:][area:enhancements]      | `area:enhancements`      | Improvements to the product that address a specific problem for users |
```

Comment:
This might be the most radical change to how we track enhancements currently, as I worry we just slap this label on everything that's "not a bug" without being more thoughtful and explicit about proposed improvements to the product.

@billygriffin I'd specifically love your input about this change and whether this makes your life easier or harder.

Reply:
I think my biggest concern about this relates to prioritization. Currently, the labels associated with prioritization (priority 1 through 3) relate to bugs, not enhancements (although I've seen that applied inconsistently in a few cases). We don't (yet) have a concept of prioritization for enhancements, and that's definitely something I'd love to have at some point. My dream would be that people could search for the combo of `help wanted` and some sense of `high priority` and have a list of ~10 or so things they could pick from - especially @desktop/comrades.

Reply:
Let's try and sync up next week about this to see if we can figure out other ideas. I'm not strongly attached to this change.